### ISSUE TITLE
(5.0 test | io-cct) Separate PNG's or Automatic Texture Atlas creation

### ISSUE THREAD

**Issue by fatihG** [univrsal/input-overlay#140]
The tool currently seems to be set up similarly to the 4.8 version, as in certain buttons need to be set up in a particular configuration for the tool to know what graphics to use when a button is pressed or unpressed etc. 

It would be nice if graphics could be loaded up individually as separate .png images and the plugin either just uses the separate images, it it generates a texture atlas in the configuration that the plugin expects. (have all Dpad images side by side on the texture atlas for example.)

I don't know if there are apps out there that let you make texture atlases that you could make use of.
There are probably some 2d sprite tools for generating texture atlases like Tiled. But I assume any texture Atlas generator with some modifying, based on your rules (unpressed face button on top, pressed facebutton below it, etc) would do the trick.

It super nice that you made the editor, but for someone that is new to the plugin/preset creation, it might not be as straight forward.

**Comment by univrsal** [comment]
That is most likely not going to happen, it's way easier to just have them all in one texture. People who want to create custom overlays will have to deal with the creation of the textures anyways so arranging them in a certain way should be the least of their problems. Splitting them up into multiple files would make the process way more complicated than it has to be.

The editor is supposed to make the creation of config files easier not the textures.

**Comment by fatihG** [comment]
Yeah I get what you are saying. For me it doesnt matter either way, just trying to think of ways to make this more user friendly.

Someone that downloads this tool will be under the impression that it will allow them to set up the entire overlay.
So you should somehow convey before hand that the texture atlas needs to be set up in a particular way.

Even if it is just a simple splash image one start up (with a checkbox to not show it anymore) that explains briefly how to set up the texture. (if you need help creating such a splash image like that just let me know.)

**Comment by univrsal** [comment]
Maybe I'll add something, although from my experience I could have users copy the info message word by word before letting them use the tool and I'd still have some come to me and explain to them how it works.

**Comment by fatihG** [comment]
Yeah, I know what you mean.
But at least then, you can be like: 'take a look at the splash image'.

Or make them open the Help Menu> Texture setup or whatever.

**Comment by univrsal** [comment]
I've added a button to the setup screen which opens the wiki page for io-cct in a browser, it's better than nothing for now I guess
